### PR TITLE
Technical review / proofreading of episode 2 with Camille Baldock from 0:45 to 1:00

### PR DESCRIPTION
**Pull request by nicoder**

### CODE REVIEW

File: 2/camille-baldock-captions.srt [hunk 1]
```diff
@@ -3572,20 +3577,20 @@ it means there happens to be
 
 823
 00:53:57,420 --> 00:53:59,790
-So if we have 'has 4', then this is
+So if we have 'has_four', then this is
```
Comment (nelstrom) [inline 2/camille-baldock-captions.srt]: Can we remove the quotes around `has_four` for consistency?

File: 2/camille-baldock-captions.srt [hunk 2]
```diff
@@ -3671,19 +3677,19 @@ then start compressing later.
 
 847
 00:55:06,000 --> 00:55:07,960
-So inaudible full house
+So elsif full_house
 
 848
 00:55:10,110 --> 00:55:11,600
-then type full house.
+then :type => :full_house.
```
Comment (nelstrom) [inline 2/camille-baldock-captions.srt:3684]: Love this! Where else do you get to see a :hash: :rocket: in subtitles?

File: 2/camille-baldock-captions.srt [hunk 3]
```diff
@@ -3773,7 +3781,8 @@ and I can drive out something a bit better.
 
 872
 00:56:37,340 --> 00:56:40,420
-inaudible with the ranking but I'm quite happy with it yet.
+i had an idea with a ranking but I'm not
```
Comment (nelstrom) [inline 2/camille-baldock-captions.srt]: First-person 'i' should be upper-case: 'I'.

**Comment by nelstrom** [comment]
Great job @nicoder, thanks!

File: 2/camille-baldock-captions.srt (hunk 1 above)
Comment (nicoder) [inline 2/camille-baldock-captions.srt]: good idea, did it in commit  acb5e08

File: 2/camille-baldock-captions.srt (hunk 3 above)
Comment (nicoder) [inline 2/camille-baldock-captions.srt]: ah, sorry about that, fixed in d0c6321

**Pull request merged**